vNext/samples/ODataSample.Web/Controllers/CustomersController.cs

```diff
@@ -26,9 +26,22 @@ public IEnumerable<Customer> Get()
 
         // GET api/Customers/5
         [HttpGet("{id}")]
-        public IActionResult Get(int id)
+        public IActionResult Get(int customerId)
```

Comment:
Does this needs to be supported? You can now use an OData call similar to this: http://localhost:5000/odata/Customers(1)